docs/docsite/rst/porting_guides/porting_guide_core_2.19.rst

```diff
@@ -5,24 +5,546 @@
 Ansible-core 2.19 Porting Guide
 *******************************
 
-This section discusses the behavioral changes between ``ansible-core`` 2.18 and ``ansible-core`` 2.19.
+This section discusses the behavioral changes between ansible-core 2.18 and ansible-core 2.19.
```

Comment:
In https://docs.ansible.com/ansible/latest/dev_guide/style_guide/preferred_terms.html#preferred-terminology, I think either Ansible Core or `ansible-core` is preferred, but @samccann can probably clarify.